src/scripts/main.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+[...people].forEach(person => {
+  const row = document.createElement('tr');
 
-// write your code here
+  row.innerHTML = `<td>${person.name}</td>
+        <td>${person.sex === 'f' ? 'female' : 'male'}</td>
+        <td>${person.born}</td>
+        <td>${person.died}</td>
+        <td>${person.died - person.born}</td>
+        <td>${Math.ceil(person.died / 100)}</td>`;
```

Comment:
```suggestion
  row.innerHTML = `
      <td>${person.name}</td>
      <td>${person.sex === 'f' ? 'female' : 'male'}</td>
      <td>${person.born}</td>
      <td>${person.died}</td>
      <td>${person.died - person.born}</td>
      <td>${Math.ceil(person.died / 100)}</td>
    `;
```
It's better to use `insertAdjacentHTML` instead of `innerHTML` in cases, where you have a loop and insert more than one element.
And it would be great if you will fix your formatting